packages/utils/src/modules-sdk/internal-module-service-factory.ts

```diff
@@ -60,7 +60,7 @@ export function internalModuleServiceFactory<
     }
 
     static buildUniqueCompositeKeyValue(keys: string[], data: object) {
-      return keys.map((k) => data[k]).join("_")
+      return keys.map((k) => data[k]).join(":")
```

Comment:
Just curious, why did we do this change?

Reply:
it happen that all our id's are composed of a prefix and a suffix separated byt `_`. Here were joing with `_` and then replace `_` by `-` which break the ids

Reply:
Ah I see 👍